### ISSUE TITLE
Sample Python Code for Uploading a Glossary

### ISSUE THREAD

**Issue by DigitalProf** [DeepLcom/deepl-python#45]
This issue is related to #44, which documented the following problem as well as a bug with processing a Byte Order Mark (BOM).  The issues here is that the following code from the [DeepL Python instructions](https://github.com/DeepLcom/deepl-python/#creating-a-glossary) does not work:

with open('/path/to/glossary_file.csv', 'r') as csv_file:
    csv_data = csv_file.read()  # Read the file contents as a string
    my_csv_glossary = translator.create_glossary_from_csv(
        "CSV glossary",
        source_lang="EN",
        target_lang="DE",
        csv_data=csv_data,
    )

The code does not work as is as documented on issue #44. Needed is the parameter that identifies the glossary file as encoded with UTF-8:

with open('/path/to/glossary_file.csv', 'r', **encoding="UTF-8**") as csv_file:

**Comment by daniel-jones-deepl** [comment]
Hi @DigitalProf Mike, thanks for following up on this topic.

I tested your suggested code, but I found that with `encoding="UTF-8"` reading a file with BOM will still result in the read file contents beginning with a `\xFEFF` character. I am not sure why this works differently on your system, perhaps this behavior differs between OSes (i.e. on Windows Python will strip the BOM, but not on Unix-based systems).

However, I found the Python codec [`utf_8_sig`](https://docs.python.org/3/library/codecs.html#module-encodings.utf_8_sig) that is intended for this case. When I used `encoding="utf-8-sig"` the `read()` call did correctly strip the BOM on my system.

I will update the example in the README in the next update. Thanks!
